src/Compilers/.editorconfig

```diff
@@ -1,6 +1,6 @@
 # Project-specific settings:
 # CSharp code style settings:
 [*.cs]
-csharp_style_var_for_built_in_types = false:none
+csharp_style_var_for_built_in_types = false:warning
```

Comment:
wait.  won't this warn if you use 'var' for things like 'string'?

Reply:
Oh sorry.  i misread.  This is in the compiler's .editorconfig.  That's fine with me.